src/test/java/software/amazon/event/kafkaconnector/EventBridgeSinkConnectorIT.java

```diff
@@ -242,6 +73,10 @@ public void sendJsonRecordToKafkaReceiveFromSQS() {
             .put("sentTimestamp", LocalDateTime.now().toString())
             .put("message", "hello from kafka");
 
+    s3client.putObject(
```

Comment:
nit: please add comment why we're doing this :)

Reply:
Will do.